### PR TITLE
Create BLorient13264

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for BLorient13264. I could not identify the prayer in ms_i7. I checked Leanders edition of the Prayers of Simeon Stylites and the unidentified prayer is not part of this edition.

I made a minor change also in BAVet57, because I have discovered, that the prayer of Ephraim encoded there is to be identified with LIT4239Prayer as well.

### CODE REVIEW

File: VaticanBAV/et/BAVet57.xml [hunk 1]
```diff
@@ -56,7 +56,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            oration after the reading of the gospel (<q xml:lang="gez">ስብሐት፡ ለእግዚአብሔር፡ በሰማያት፡ ወሰላም፡ በምድር፡ ሥምረቱ፡ ለሰብእ፨ በትንብልናሃ፡ ለእግዝእተ፡ ኵልነ፡ ቅድስት፡ 
                               ማርያም፡ ወላዲተ፡ አምላክ፡ ጸግወነ፡ እግዚኦ፡ ስርየተ፡ ኃጣውኢነ፡ ንሰግድ፡ ለከ፡ ኦክርስቶስ፡ ምስለ፡ አቡከ፡ ኄር፡ ወመንፈስ፡ ቅዱስ፡</q>), a prayer entitled
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet57.xml:57]: Thank you! Ideally, this list should be transformed in 
a list of `<msItem xml:id="ms_i1.2.1">` etc, each with its own `title `and where provided `incipit` but I agree this is a lot of work, so thank you for inserting the reference so it is not lost

File: LondonBritishLibrary/orient/BLorient13264.xml [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">150</measure>
+                                    <measure unit="leaf" type="blank" corresp="#2v #66v #81v #142v #150">5</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>240</height>
+                                        <width>190</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>Seventeen ouf of eighteen quires are numbered with quiremarks.</foliation>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13264.xml:193]: quire numbering is not part of `foliation `but of `collation`

according to the TEI guidelines the best element could be `signatures `https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-signatures.html (inside `collation`)

File: LondonBritishLibrary/orient/BLorient13264.xml [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+                                <decoNote xml:id="b4" type="SlipCase">Contained in a double hide case.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1600" notAfter="1799">17th to 18th century</origDate>
+                        </origin>
+                        <provenance>According to <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/>
+                            <citedRange unit="page">9-10</citedRange></bibl> taken from King Theodore's Library in Magdala in
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13264.xml:274]: mark down  `<persName ref="PRS9430Tewodros"/>` and `<placeName ref="INS0101MadhaneAlam"/>`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient13264.xml [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="111ra" to="125rb"/>
+                            <title ref="LIT3179Revela"/>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <locus from="126ra" to="133va"/>
+                            <title ref="LIT4872MagdemaWangel">Introduction to the Gospels</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ንቅድም፡ በረድኤተ፡ እግዚአብሔር፡ ሎቱ፡ ስብሐት፡ 
+                                <supplied reason="omitted" resp="Strelcyn">ይትባረክ፡ ስሙ፡</supplied> ወይትለዓል፡ ዝክሩ፡ ወፍጽምናሁ፡ ዘእምኔሁ፡ ሢመተ፡ ኵሉ፡
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient13264.xml:156]: I think the PRS ID has to be used in resp

File: LondonBritishLibrary/orient/BLorient13264.xml [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">150</measure>
+                                    <measure unit="leaf" type="blank" corresp="#2v #66v #81v #142v #150">5</measure>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient13264.xml:187]: page

File: LondonBritishLibrary/orient/BLorient13264.xml [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+                                <q xml:lang="gez">ዘሥራይ፡ መድ<supplied reason="omitted" resp="Strelcyn">ኀኒት</supplied>፡ </q>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="65ra" to="66ra"/>
+                                    <title ref="LIT2940RepCh221"/>
+                                    <desc type="GuestText"/>
+                                    <q xml:lang="en">ሰላም፡ ሰላም፡ ለዝክር፡ ስምኪ፡ በአምሐ፡ </q>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient13264.xml:221]: gez

File: LondonBritishLibrary/orient/BLorient13264.xml [hunk 7]
```diff
@@ -0,0 +1,329 @@
...
+                                        </roleName>Gabra Ḥǝywat</persName> and his son 
+                                        <persName ref="PRS14336Yohannes">Yoḥannǝs</persName>.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1"/>
+                                <decoNote type="bindingMaterial" xml:id="b2"><material key="wood"/></decoNote>
+                                <decoNote xml:id="b3" type="Cover"><material key="leather">Stamped leather.</material></decoNote>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient13264.xml:264]: blind-tooled

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

**Review by thea-m — APPROVED** [review]

**Pull request merged**